### PR TITLE
Resolved Bug 172 : Storybook > Component > Account > Default > Captcha > Issues on Account component Small, Large mobile view captcha not visible.

### LINKED ISSUE #172: Storybook > Component > Account > Default > Captcha > Issues on Account component Small, Large mobile view captcha not visible.

### PR DESCRIPTION
**Pull request by poojagurav21**
-Navtabs -selected tab text color issue resolved
-required message not clear after save-issue resolved
-Version select list not clear after save-issue resolved

## Screenshots :
image
image
image

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: raaghu-components/src/rds-comp-account/rds-comp-account.tsx [hunk 1]
```diff
@@ -55,6 +81,7 @@ const RdsCompAccount = (props: RdsCompAccountProps) => {
 
     function emitSaveData(event: any) {
         event.preventDefault();
+        console.log("Before Save:", accountGeneralData);
```
Comment (hemalatagaikwad) [inline raaghu-components/src/rds-comp-account/rds-comp-account.tsx]: remove console.log

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
remove cosole.log statement

File: raaghu-components/src/rds-comp-account/rds-comp-account.tsx (hunk 1 above)
Comment (poojagurav21) [reply raaghu-components/src/rds-comp-account/rds-comp-account.tsx]: Removed

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Pull request merged**